src/pyOpenMS/pxds/ContactPerson.pxd

```diff
@@ -14,41 +14,41 @@ cdef extern from "<OpenMS/METADATA/ContactPerson.h>" namespace "OpenMS":
         ContactPerson(ContactPerson) nogil except + # wrap-ignore
 
         # returns the first name of the person
-        String getFirstName() nogil except +
+        String getFirstName() nogil except + # wrap-doc:Returns the first name of the person
         # sets the first name of the person
-        void setFirstName(String name) nogil except +
+        void setFirstName(String name) nogil except + # wrap-doc:Sets the first name of the person
 
         # returns the last name of the person
-        String getLastName() nogil except +
+        String getLastName() nogil except + # wrap-doc:Returns the last name of the person
         # sets the last name of the person
-        void setLastName(String name) nogil except +
+        void setLastName(String name) nogil except + # wrap-doc:Sets the last name of the person
 
         # sets the full name of the person (gets split into first and last name internally)
-        void setName(String name) nogil except +
+        void setName(String name) nogil except + # wrap-doc:Sets the full name of the person (gets split into first and last name internally)
 
         # returns the affiliation
-        String getInstitution() nogil except +
+        String getInstitution() nogil except + # wrap-doc:Returns the affiliation
         # sets the affiliation
-        void setInstitution(String institution) nogil except +
+        void setInstitution(String institution) nogil except + # wrap-doc:Sets the affiliation
 
         # returns the email address
-        String getEmail() nogil except +
+        String getEmail() nogil except + # wrap-doc:Returns the email address
         # sets the email address
-        void setEmail(String email) nogil except +
+        void setEmail(String email) nogil except + # wrap-doc:Sets the email address
 
         # returns the email address
-        String getURL() nogil except +
+        String getURL() nogil except + # wrap-doc:Returns the email address
```

Comment:
Copy paste?

Reply:
I got the same comment in the header file. Any suggestions for improvement?

Reply:
Returns the URL associated with the contact person (e.g., the institute webpage "https://www.higglesworth.edu/")

Reply:
please add Timos suggestion